dom0/sd-dom0-files.sls

```diff
@@ -46,31 +46,11 @@ dom0-workstation-rpm-repo:
     - require:
       - file: dom0-rpm-test-key
 
-dom0-workstation-templates-repo:
-  # Using file.blockreplace because /etc/qubes/repo-templates/ is not a .d
-  # style directory, and qvm.template_installed:fromrepo seems to only support
-  # using a repo from this file. Installing manually via a cli-command-instead?
-  file.blockreplace:
-    - name: /etc/qubes/repo-templates/qubes-templates.repo
-    - append_if_not_found: True
-    - marker_start: "### BEGIN securedrop-workstation ###"
-    - marker_end: "### END securedrop-workstation ###"
-    - content: |
-        [securedrop-workstation-templates]
-        gpgcheck=1
-        gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation
-        enabled=1
-        baseurl={{ sdvars.dom0_yum_repo_url }}
-        name=SecureDrop Workstation Templates repository
-    - require:
-      - file: dom0-rpm-test-key
-
-dom0-install-securedrop-workstation-template:
+# Ensure debian-12-minimal is present for use as base template
+dom0-install-debian-minimal-template:
   cmd.run:
```

Comment:
I _think_ we can just use `[qvm.template_installed](https://github.com/QubesOS/qubes-mgmt-salt-dom0-qvm?tab=readme-ov-file#qvmtemplate_installed)` instead of `cmd.run`.  If memory serves, we had an issue with the qvm-template tooling because we were using the same repo to serve templates and the dom0 config rpm, and qvm-template was upset by this.

Reply:
This can probably be an issue.